modules/overview/pages/introduction-to-eclipse-che.adoc

```diff
@@ -55,7 +55,7 @@ Learn about the latest of {prod} and submit your blog posts to the link:https://
 
 Weekly meetings::
 
-Join us in the link:https://github.com/eclipse/che/wiki/{prod-short}-Dev-Meetings[{prod-short} community meeting] every Monday.
+Join us in the link:https://github.com/eclipse/che/wiki/{prod-id}-community-meetings[{prod-short} community meeting] every Monday.
```

Comment:
Community call is on demand starting from August  - https://github.com/eclipse-che/che/wiki/Eclipse-Che-Community-Meetings#when-and-where

Reply:
Good point, I changed it